packages/sdk/test/zora-fork-mint.test.ts

```diff
@@ -79,33 +79,32 @@ describe.skipIf(!process.env.VITE_ALCHEMY_API_KEY)(
       // This is a workaround to this known issue: https://github.com/NomicFoundation/hardhat/issues/5511
       await mine();
 
-      // Step defining the action for Transfer event
+      // Step defining the action for Purchased event
       const eventActionStep: ActionStep = {
-        signature: selector, // Transfer(address,address,uint256) event signature
+        chainid: base.id,
+        signature: selector, // Purchased(address,address,uint256,uint256,uint256) event
         signatureType: SignatureType.EVENT, // We're working with an event
         actionType: 0, // Custom action type (set as 0 for now)
-        targetContract: targetContract, // Address of the ERC20 contract
-        // We want to target the Minter property on the Purchase event
+        targetContract: targetContract, // Address of the zora NFT contract
+        // We want to target the 'sender' property on the Purchase event
         actionParameter: {
           filterType: FilterType.EQUAL, // Filter to check for equality
           fieldType: PrimitiveType.ADDRESS, // The field we're filtering is an address
-          fieldIndex: 1, // Might need to be 2, we'll see - let's log this
-          filterData: boostImpostor, // Filtering based on the core address
+          fieldIndex: 1, // The sender is at the 1st topic on the event
+          filterData: boostImpostor, // Filtering based on the imposters address
         },
       };
 
       // Define EventActionPayload manually
       const eventActionPayload = {
         actionClaimant: {
+          chainid: base.id,
           signatureType: SignatureType.EVENT,
-          signature: selector, // Transfer(address,address,uint256) event signature
+          signature: selector, // Purchased(address,address,uint256,uint256,uint256) event
           fieldIndex: 0, // Targeting the 'from' address
           targetContract: boostImpostor, // The ERC20 contract we're monitoring
```

Comment:
This does not seem right. The boostImposter is an EOA. Should this not be the zora NFT contract (targetContract)?

Reply:
@Quazia ?